packages/browser-repl/src/components/editor.tsx

```diff
@@ -1,10 +1,9 @@
 import React, { Component } from 'react';
-import AceEditor from 'react-ace';
-import type { IAceEditor } from 'react-ace/lib/types';
 import 'ace-builds';
-import 'ace-builds/webpack-resolver';
 import tools from 'ace-builds/src-noconflict/ext-language_tools';
 import 'ace-builds/src-noconflict/mode-javascript';
+import type { IAceEditor } from 'react-ace/lib/types';
+import AceEditor from 'react-ace';
```

Comment:
Everything I've experienced and read in the issues on their repo points at it being better to only import react-ace _after_ ace-builds.